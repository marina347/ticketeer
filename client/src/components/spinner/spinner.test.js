import React from "react";
import { shallow } from "enzyme";
import Spinner from "./spinner.component";

describe("Spinner", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Spinner />);
  });

  it("Should render Spinner", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
