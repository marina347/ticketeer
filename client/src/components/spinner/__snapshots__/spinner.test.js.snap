// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Should render Spinner 1`] = `
<div
  class="spinner-container"
>
  <div
    class="spinner"
  />
</div>
`;
